### PR TITLE
Backend lambda

### PR DESCRIPTION
**Pull request by hnpope**
Lambdas finalized for release 1

### CODE REVIEW

File: dummyLambda.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+lsdef lambda():
```
Comment (cspatil8) [inline dummyLambda.py:1]: Don't commit this file to master

File: lambdas/EmailNotification.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            Source=contentTuple[0],
+            Destinations=[
+                contentTuple[1]
+            ],
+            RawMessage={
+                'Data':contentTuple[2],
+            },
+            #ConfigurationSetName=CONFIGURATION_SET
+        )
+        print('NO ERROR IN SENDING EMAIL')
```
Comment (cspatil8) [inline lambdas/EmailNotification.py:92]: 'EMAIL SENT SUCCESSFULLY'

File: lambdas/cancel_appointment.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    cancel_reason = event['cancel_reason']
+    #Connect to table
+    client = boto3.client('rds-data')
+
+    #Check if appointment exists
+    existing_user = client.execute_statement(
+        secretArn = "arn:aws:secretsmanager:us-east-2:500514381816:secret:rds-db-credentials/cluster-33FXTTBJUA6VTIJBXQWHEGXQRE/postgres-3QyWu7", 
+        database = "postgres",
+        resourceArn = "arn:aws:rds:us-east-2:500514381816:cluster:postgres",
+        sql = "SELECT appointment_id FROM scheduled_appointments WHERE appointment_id = '%d';" % (appointment_id_to_delete)
```
Comment (cspatil8) [inline lambdas/cancel_appointment.py:16]: Use the query_db.py from /lambda/package

File: lambdas/cancel_appointment.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        return{
+            'statuscode' : 404
+        }
+
+    cancel_bool = 'True'
+    #This is the query that we need to execute
+    updateQuery = "UPDATE scheduled_appointments SET cancelled = '%s', cancel_reason = '%s' WHERE appointment_id = '%d';" % (cancel_bool, cancel_reason, appointment_id_to_delete)
+
+    #Accessing database and executing query
+    client.execute_statement(
```
Comment (cspatil8) [inline lambdas/cancel_appointment.py:29]: Same^

File: lambdas/package/query_db.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+from package.lambda_exception import LambdaException
+from package import db_config
+import boto3
+
+secretArn = db_config.SECRET_ARN
+dbName = db_config.DB_NAME
+arn = db_config.ARN
+
+def query(sql_query):
```
Comment (cspatil8) [inline lambdas/package/query_db.py:9]: Add for args and return type

File: lambdas/create_appointment.py [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    location = event['location']
+
+    # connect to db
+    client = boto3.client('rds-data')
+
+    query = f"insert into scheduled_appointments 
+                (supporter_id, time, type, duration, method, cancelled, cancel_reason, location) 
+                values ({supporter_id}, {time}, {appt_type}, {duration}, {"In Person"}, {False}, {None}, {location})"
+    
+    response = client.execute_statement(
```
Comment (cspatil8) [inline lambdas/create_appointment.py:25]: Use query_db.py

**Comment by dergoose** [comment]
prolly wanna fix that formating in your git ignore

**Pull request closed without merging**